_source/_posts/2018-06-28-tutorial-build-a-basic-crud-app-with-node.md

```diff
@@ -196,18 +171,17 @@ app.use(express.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, "public")));
 
 const oidc = new ExpressOIDC({
-  issuer: "{yourOktaOrgUrl}/oauth2/default",
-  client_id: "{yourOktaClientId}",
-  client_secret: "{yourOktaClientSecret}",
-  redirect_uri: "http://localhost:3000/users/callback",
+  appBaseUrl: "http://localhost:3000",
+  issuer: "{OKTA_OAUTH2_ISSUER}",
```

Comment:
I believe the standard convention for issuer is `https://{yourOktaDomain}/oauth2/default`, so IMO I think the issuer, client_id, and client_secret LOC need to be reverted to previous post convention.

Reply:
Since the Okta CLI gives the issuer URL, wouldn't it be easier for readers to copy-paste that vs. copy a part of that URL and paste it in the middle of that string? :)

Reply:
i hear what you're saying and can see how it's helpful, but the prior text is consistent with other posts. I don't feel comfortable making the call away from consistency on my own so maybe team discussion on Slack?